README.md

```diff
@@ -0,0 +1,107 @@
+# wzdx_registry
+
+Code for triggering lambda function for ingesting WZDx feed data based on the WZDx Feed Registry Socrata dataset.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+### Prerequisites
+
+#### Prerequisites for AWS Lambda Deployment
+
+If you plan to deploy the script on AWS Lambda, you need access to an AWS account and be able to assign role(s) to a lambda function. There needs to be a role that is able to execute and invoke lambda functions and perform list/read/write actions to relevant buckets in S3.
+
+#### Prerequisites for Local Deployment
+
+If you plan to deploy the script on your local machine, you need the following:
+
+1. Have access to Python 2.7 or Python 3.6+. You can check your python version by entering `python --version` and `python3 --version` in command line.
```

Comment:
Good point - didn't realize Python 2 would be officially unsupported by 1/1/2020. We can just aim to support Python 3.6+ then - shouldn't be a problem since the code is meant to be deployed via AWS Lambda.

Reply:
Sounds good. Maybe we can reconsider next year.